.github/workflows/renovate-deploy.yml

```diff
@@ -22,14 +22,20 @@ jobs:
           repositories=$(echo "${{ github.event.inputs.deployRepository }}" | cut -d '/' -f2)
           echo "owner=$owner" >> "$GITHUB_OUTPUT"
           echo "repositories=$repositories" >> "$GITHUB_OUTPUT"
+      - name: Load Secrets from Vault
+        if: steps.override.outputs.use-gh-app == 'true'
+        uses: rancher-eio/read-vault-secrets@main
+        with:
+          secrets: |
+            secret/data/github/org/${{ github.repository_owner }}/github/renovate-rancher appId | APP_ID ;
+            secret/data/github/org/${{ github.repository_owner }}/github/renovate-rancher privateKey | PRIVATE_KEY
       - name: Get token
+        if: steps.override.outputs.use-gh-app == 'true'
         id: get_token
         uses: actions/create-github-app-token@df432ceedc7162793a195dd1713ff69aefc7379e # v2.0.6
         with:
-          app-id: ${{ secrets.RENOVATE_APP_ID }}
-          private-key: ${{ secrets.RENOVATE_PRIVATE_KEY }}
-          owner: ${{ steps.set_values.outputs.owner }}
-          repositories: ${{ steps.set_values.outputs.repositories }}
```

Comment:
These 2 lines must remain, because we are generating the GH App token against the repo (`owner/repository`) that was defined when the user triggered the workflow.